package.json

```diff
@@ -1,16 +1,16 @@
 {
   "name": "tattoo",
-  "version": "1.0.2",
+  "version": "2.0.0-alpha",
   "description": "Test all the things over and over",
-  "main": "lib/tattoo.js",
+  "main": "lib/runner.js",
   "bin": {
-    "tattoo": "lib/tattoo.js"
+    "tattoo": "lib/bin/tattoo.js"
   },
   "scripts": {
-    "format": "find src | grep '\\.ts$' | xargs clang-format --style=file -i",
-    "test": "tslint *.ts custom_typings/*.ts",
+    "format": "find src custom_typings | grep '\\.ts$' | xargs clang-format --style=file -i",
+    "test": "tslint src/*.ts src/**/*.ts custom_typings/*.ts; mocha",
```

Comment:
It totally should, but I ran this on stock bash installed on my Mac and `shopt -q globstar` isn't available until bash 4 so ** was meaningless.  Technically it should just read `src/**.ts` so this is wrong for two reasons.

Removed the double star in 40de3bdd5a2b097b5cadfe35539589ecc13ecd9f